__init__.py

```diff
@@ -52,6 +53,7 @@ def __init__(self, *args, **kwargs):
             'Scrabble score',  # spammy
             'Other notable uses'  # spammy
         ]
+        self.initialize()
```

Comment:
if this isnt called its a serious bug in core, it def should not be called manually

Reply:
self.wolfie was None until I did this. Not sure if maybe a different exception was raised and swallowed but this got the code working on the Mark 1.

Reply:
That’s weird. Surely it’d happen to every skill and not just Wolfie. Have you guys tried diffing the boilerplate part against another skill?

Reply:
Oh, I see, it was using the old imports. This might be a hassle to chase down.

To clarify, was it None when it imported from Mycroft.skill, after the change to ovos.util, or both?

Reply:
Both, so far as I could tell. I haven't done any testing to back this up yet, but do we know for sure that this isn't a regression in core? I've seen a lot of skills just not working properly recently. Of the default skills in the RaspOvos image, how many are still using the Mycroft-style initialize?

Reply:
```
ovos-skill-date-time                 0.2.3a5 initialize
ovos-skill-fallback-unknown          0.0.4a1 no initialize
ovos-skill-hello-world               0.0.4a3 no initialize
ovos-skill-jokes                     0.0.1 **Couldn't find**
ovos-skill-naptime                   0.2.3a1 initialize, also not working as expected
ovos-skill-personal                  0.0.4a2 no initialize
ovos-skill-somafm                    0.0.2 no initialize
ovos-skill-stop                      0.3.0a4 no initialize
ovos-skill-volume                    0.0.2a3 no initialize
ovos-skill-weather                   0.0.1a13 initialize
```

Reply:
Again, no data to back it up yet, but the skills I've seen with problems right now use both `self.initialize()` and have a `create_skill()` function. The rest don't. That's probably where I'll start looking for additional data points.